### ISSUE TITLE
[capstone2llvmir]: Missing type for register number: 245

### ISSUE THREAD

**Issue by foiso001** [avast/retdec#1160]
I am trying to lift an andoird kernel binary but I am facing the following error:

```
Missing type for register number: 51204
storeRegister() unhandled reg.
Missing type for register number: 51204
storeRegister() unhandled reg.
[capstone2llvmir]: Missing type for register number: 245
```

Is there any direct way to fix this? Also how can I locate the missing registers and add the definition to the tool?

**Comment by Nitr0-G** [comment]
> I am trying to lift an andoird kernel binary but I am facing the following error:
> 
> ```
> Missing type for register number: 51204
> storeRegister() unhandled reg.
> Missing type for register number: 51204
> storeRegister() unhandled reg.
> [capstone2llvmir]: Missing type for register number: 245
> ```
> 
> Is there any direct way to fix this? Also how can I locate the missing registers and add the definition to the tool?

https://github.com/avast/retdec/issues/1148
We need to fix a little for Android as well...

**Comment by foiso001** [comment]
@Nitr0-G Hi there, so I need to debug the information for the registers and check which bytes are not translated correctly, right? I will give a thorough read of your fix and get back to you. Thanks for the suggestions!

**Comment by Nitr0-G** [comment]
> @Nitr0-G Hi there, so I need to debug the information for the registers and check which bytes are not translated correctly, right?

Yes, you're right, and also say the names of these registers. Then I can easily remove this problem. I could do it myself, but I've never had anything to do with Android.
